### ISSUE TITLE
Jigasi quits the conference when user with owner permissions leaves the room

### ISSUE THREAD

**Issue by paweldomas** [jitsi/jigasi#13]
Jigasi quits the conference when user with owner permissions leaves the room. We should reconsider this behaviour and make it leave only when call initiator leaves or Jicofo's user leaves(or remove this logic completely).

**Comment by dosedawa** [comment]
Indeed I have been able to reproduce it, it would be great if we can leave them in the conference within disconnecting all people who joined by jigasi!

**Comment by nitrado-leo** [comment]
Very annoying bug/feature. Everyone here logs in as admin, SIP call is dropped all the time.

**Comment by nitrado-leo** [comment]
This is where it happens: https://github.com/jitsi/jigasi/blob/2fc11848569e40373c2500c98a726f049ec36646/src/main/java/org/jitsi/jigasi/JvbConference.java#L735

This logic should probably be limited to the focus user, since there are plenty of valid use cases where an owner user establishes a SIP call, but leaves afterwards for whatever reason.

**Comment by paweldomas** [comment]
Yes, that's where it happens. Jigasi is not currently our priority, but a PR which fixes the issue would be welcome.

**Comment by paweldomas** [comment]
Actually this might be a one line fix, but would have to think about it.

**Comment by nitrado-leo** [comment]
It's indeed a one line fix, I removed the owner user condition and it works as expected. Should I make a PR which makes it configurable? Or remove it altogether?

Reasoning:

- The check does not take into account conferences with multiple owners. Some of our users have spotty network coverage and have to reconnect periodically, kicking all Jigasi participants from the conference (which makes it unusable).
- The owner has the option of manually disconnecting the before leaving. Owner permissions are required for making new SIP calls, so the remaining non-owner participants can't do anything dangerous even if the call stays connected. Am I missing something?

Jigasi is extremely useful for us, by the way. Apart from this, it works just fine. Thanks!
